corehq/apps/app_manager/templates/app_manager/managed_app.html

```diff
@@ -148,6 +148,14 @@ <h2 class="appnav-name">
             </button>
           </div>
         {% endif %}
+        {% if show_training_modules %}
+          <div class="pull-left">
+            <button type="button" class="popover-additem-option new-module" data-type="training">
+              <i class="fa fa-graduation-cap"></i> {% trans "Training" %}
+              <p>{% blocktrans %}Updates for trainers.{% endblocktrans %}</p>
```

Comment:
Love the icon, but I think the help text doesn't quite convey the right thing. My suggestion would be `Content for training users.`

Reply:
Updated:
<img width="663" alt="screen shot 2018-04-05 at 11 41 23 am" src="https://user-images.githubusercontent.com/1757035/38376333-53ce1058-38c6-11e8-9809-cab41f82e5c1.png">